compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -165,6 +166,31 @@ extern "C" LLVMValueRef LLVMRustGetNamedValue(LLVMModuleRef M, const char *Name,
   return wrap(unwrap(M)->getNamedValue(StringRef(Name, NameLen)));
 }
 
+enum class LLVMRustVerifierFailureAction {
+    AbortProcessAction = 0,
+    PrintMessageAction = 1,
+    ReturnStatusAction = 2,
+};
+
+static LLVMVerifierFailureAction fromRust(LLVMRustVerifierFailureAction Action) {
+  switch (Action) {
+    case LLVMRustVerifierFailureAction::AbortProcessAction:
+      return LLVMAbortProcessAction;
+    case LLVMRustVerifierFailureAction::PrintMessageAction:
+      return LLVMPrintMessageAction;
+    case LLVMRustVerifierFailureAction::ReturnStatusAction:
+      return LLVMReturnStatusAction;
+  }
+  report_fatal_error("Invalid LLVMVerifierFailureAction value!");
+}
+
+//    pub fn LLVMRustVerifyFunction(V: &Value, action: LLVMRustVerifierFailureAction) -> Bool;
+extern "C" bool LLVMRustVerifyFunction(LLVMValueRef Fn,
+                                       LLVMRustVerifierFailureAction Action) {
+  //Function *F = unwrap<Function>(Fn);
+  return LLVMVerifyFunction(Fn, fromRust(Action));
+}
```

Comment:
Uhh, sorry, but looks like after rebase this reverted back to wrong LLVMBool\bool types. And stray commendted out code.

Reply:
Thank you, I also formated the c++ file.